skymp5-server/cpp/server_guest_lib/PieScript.cpp

```diff
@@ -2,16 +2,15 @@
 
 #include "FormDesc.h"
 #include "MpActor.h"
+#include "NiPoint3.h"
 #include "SpSnippet.h"
 #include "WorldState.h"
 #include <random>
 #include <sstream>
-#include <stdexcept>
-#include <vector>
-
+#include <stdexcept> #include <unordered_map> #include <vector>
 std::mt19937 g_rng{ std::random_device{}() };
 
-int GenerateRandomNumber(int leftBound, int rightBound)
+uint32_t GenerateRandomNumber(uint32_t leftBound, uint32_t rightBound)
 {
   if (leftBound <= rightBound) {
     std::uniform_int_distribution<> distr(leftBound, rightBound);
```

Comment:
This probably causes implicit uint -> int conversion. Probably not going to cause problems though, but still... I guess that's ok if it was being converted before